<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateSelectorPanel</class>
 <widget class="QWidget" name="TemplateSelectorPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>549</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListView" name="listView"/>
   </item>
  </layout>
  <action name="actionNew">
   <property name="text">
    <string>New Template</string>
   </property>
   <property name="toolTip">
    <string>create new template</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete Template</string>
   </property>
   <property name="toolTip">
    <string>delete the selected template</string>
   </property>
  </action>
  <action name="actionSet_Default">
   <property name="text">
    <string>Set Default</string>
   </property>
   <property name="toolTip">
    <string>set the current template default</string>
   </property>
  </action>
  <action name="actionUnset_Default">
   <property name="text">
    <string>Unset Default</string>
   </property>
   <property name="toolTip">
    <string>unset the current template default</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
